### PR TITLE
Add lektor plugins to Readme

### PR DESCRIPTION
**Pull request by rlaverde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,3 +56,11 @@ Webpack is used for built and generate static files (css, js, images, fonts), in
 - .scss files will be processed with Sass
 - .js files will be processed with Babel to convert ES6 into ES5
 - JS and CSS files will be minified
+
+### Lektor plugins
+
+The built also requiere several [Lektor Plugins](https://www.getlektor.com/docs/plugins/) (No extra configuración is needed, Lektor automatically install them when building):
```
Comment (goanpeca) [inline README.md]: The build.... also requires...

**Comment by goanpeca** [comment]
Minor style comments, otherwise looks good :-)

**Comment by rlaverde** [comment]
> Minor style comments

Fixed

**Comment by goanpeca** [comment]
I dont see the fixes

**Comment by rlaverde** [comment]
> I dont see the fixes

I amend the commit, and force push it :/

**Comment by goanpeca** [comment]
Dont know what you did

image

But the errors are there, please fix and push a new commit.

**Comment by goanpeca** [comment]
**The build also requires**

You have

**The built also requieres**

**Review by goanpeca — APPROVED** [review]

**Pull request merged**